frontend/src/metabase/visualizations/components/LineAreaBarChart.jsx

```diff
@@ -273,9 +273,14 @@ export default class LineAreaBarChart extends Component {
     const canSelectTitle = cardIds.size === 1 && onChangeCardAndRun;
 
     const hasMultipleSeries = series.length > 1;
+    const hasMultipleDimensions = settings["graph.dimensions"].length > 1;
```

Comment:
We usually call this `hasBreakout`

Reply:
You'd think I'd know that by now 😂. I'll update it